.circleci/config.yml

```diff
@@ -67,7 +67,7 @@ jobs:
       - run: pre-commit run --all-files
 
       # verify that the testing.TestingT interface is used instead of testing.T
-      - run: grep -Rw "*testing\.T" modules | grep -v _test.go
+      - run: [[ $(grep -Rw "*testing\.T" modules | grep -v _test.go | wc -l) -eq "0" ]]
```

Comment:
I hope you don't mind me joining the review. it's great to see the work done enabling Terratest and gingko to work together!

Would it be better to add this check to the pre-commits as a local test? it means the check can happen prior to reaching CI

**Example (untested)**
```
- repo: local
  hooks:
    - id: test-interfaces-used
      name: test-interfaces-used
      entry: bash -c 'grep -Rw "*testing\.T" modules | grep -v _test.go | wc -l'
      language: system
      types: [go]
      pass_filenames: false
```

Reply:
That seems like a good idea!